lib/modules/dosomething/dosomething_reportback/includes/Reportback.php

```diff
@@ -170,11 +170,11 @@ private function build($data, $full = false) {
 
     $this->user = [
       'drupal_id' => $data->uid,
-      'id' => dosomething_helpers_isset($northstar_user, 'id'),
-      'first_name' => dosomething_helpers_isset($northstar_user, 'first_name'),
-      'last_initial' => dosomething_helpers_isset($northstar_user, 'last_initial'),
-      'photo' => dosomething_helpers_isset($northstar_user, 'photo'),
-      'country' => dosomething_helpers_isset($northstar_user, 'country'),
+      'northstar_id' => dosomething_helpers_isset($northstar_user->id, 'id'),
+      'first_name' => dosomething_helpers_isset($northstar_user->first_name, 'first_name'),
+      'last_initial' => dosomething_helpers_isset($northstar_user->last_initial, 'last_initial'),
+      'photo' => dosomething_helpers_isset($northstar_user->photo, 'photo'),
+      'country' => dosomething_helpers_isset($northstar_user->country, 'country'),
```

Comment:
These should be breaking... for the `dosomething_helpers_isset()`, if it's an object, you pass it in and then indicate the property as a string, like `dosomething_helpers_isset($northstar_user, 'first_name')`... you don't pass the object and arrow property `$northstar_user-> first_name `